py/examples/device_tapdetector_example/README.md

```diff
@@ -0,0 +1,101 @@
+# Tap Detector
+
+## About
+The tap detector is designed to take input from two different sources of data and to "match" them together by detecting taps between the two devices. For example, two devices recording at the same time can be tapped against each other. Then, the resulting data files from both devices can be used with the tap detector to find the corresponding taps in either file so that their timestamps can be aligned. Data smoothing is done using a hann filter, whose window is specified by the user for each source of data.
+
+Within this directory, there are three examples:
+
+```tapdetector_example.py``` which shows how to use the tapDetector in full.
+
+```tapdetector_extractdata_example.py``` which shows how to use the individual ```extract_data()``` function included.
+
+```tapdetector_loaddata_example.py``` which shows how to use the individual ```load_data()``` function included.
+
+The tapdetector was written and tested using a specific anaconda environment, see the quick start example for more information on finding and using this environment.
+
+## Usage
+
+### Quick start example using sample data
+There is sample data to use with the tapDetector provided in ```ExampleDataForTapDetector.zip```. This section will show you how to use that data in the tapDetector.
+
+After extracting the data from the .zip file, ensure that all the sample data files are in the same working directory as the ```tapdetector.py``` file. Then, ensure that you have enabled the ```EmotiBit-pyenv-scipy``` anaconda environment. This environment can be found in the ```EmotiBit-pyenv.yml``` file found [in EmotiBit_Biometric_Lib/py/anaconda-environments/](https://github.com/EmotiBit/EmotiBit_Biometric_Lib/tree/master/py/anaconda-environments).
+
```

Comment:
Another note, using `py` did not work for me.
![image](https://github.com/EmotiBit/EmotiBit_Biometric_Lib/assets/31810812/6ff1fbdc-dfb1-4d74-baed-966a1cf1884c)

I had to use `python`

Reply:
Also, the script saves plots and `*Taps.csv` in the calling directory.
All the more reason to not call it from `py/emotibit`

![image](https://github.com/EmotiBit/EmotiBit_Biometric_Lib/assets/31810812/2be66ef7-c1cc-4743-a505-93cd536ca061)

Reply:
- Changed the link to the anaconda folder to be relative
- Changed directions to instruct to use the python command, but also noted the existence of py in case other users have issues with python
- Changed directions to not instruct user to put sample data and working directory in the same place as the script.